.vsts-ci.yml

```diff
@@ -45,7 +45,8 @@ jobs:
         PYTHON_VERSION: '3.7'
       bdist:
         TASK: bdist
-        PYTHON_VERSION: '3.8'
+        # on Ubuntu 14.04, test_dual.py fails with newer version of Python
+        PYTHON_VERSION: '3.7'
```

Comment:
Could you please share the exact error message?

Reply:
This is the same error that we get when trying to do gpu-source with Python 3.8 on Ubuntu 14.04